src/main/resources/lang/en-US.yml

```diff
@@ -106,7 +106,6 @@ COMMAND_ARGUMENT_ENTITY: 'entity'
 COMMAND_ARGUMENT_ISLAND: 'island'
 COMMAND_ARGUMENT_ISLAND_NAME: 'island-name'
 COMMAND_ARGUMENT_ISLAND_ROLE: 'island-role'
-COMMAND_ARGUMENT_ISLAND_WARP 'island-warp'
```

Comment:
Is there a reason why you removed this line?

Reply:
Probably a mistake